### PR TITLE
Move hatching to pretransition

### PR DESCRIPTION
**Pull request by eileenprice**
Eliminates the "flash" of non-hatched bars on the column chart and on most chart not available views.

### CODE REVIEW

File: addon/components/base-chart-component.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default Component.extend({
     resizeTimer: null,
     onResizeDebounced: null,
     isChartAvailable: true,
-    chartNotAvailableMessage: '',
+    chartNotAvailableMessage: 'Chart not available for this view',
```
Comment (JackieChiles) [inline addon/components/base-chart-component.js]: I don't think we can have this default here as it will not be localized

Comment (Jbcampbe) [reply addon/components/base-chart-component.js]: We always override this property and pass in a localized version in EEQ though correct?

Comment (JackieChiles) [reply addon/components/base-chart-component.js]: Yes, but if it was not passed in we'd have this non-localized string

Comment (eileenprice) [reply addon/components/base-chart-component.js]: Wouldn't a default "this chart isn't available" string be better than no text at all, though?
Similar to the way colors are default if we don't pass them--we ALWAYS pass colors, but on the off chance we don't, we specify a default that isn't dc's default colors

**Pull request merged**